common/ontology/homelab.owl

```diff
@@ -0,0 +1,475 @@
+<?xml version="1.0"?>
```

Comment:
Could you perhaps change/shorten the URL to something more meaningful? Maybe `http://b-it-bots.de/ontologies/homelab.owl` or something like that?

Reply:
I have changed the url to `http://b-it-bots.de/ontologies/homelab.owl`